### DISCUSSION TITLE
[Feature Request] Delete open message and return to current folder overview

### DISCUSSION THREAD

**Discussion by nickth76** [tutao/tutanota#3432] [Ideas]
Dear Tutanota developers, 

I'm writing this to propose a change in current function of the app, at least the mobile app. 

I think it would be better if when deleting an open message the app returns to the current folder overview, rather than automatically open the next one.  
An option so that the user can choose what behavior she/he wants would be a good addition.

Thanks in advance.

Regards
Nick

**Comment by charlag** [comment]
I think in especially in single column layout it doesn't make too much sense to open the next open and I wouldn't mind a setting but we would need to discuss it
